amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H

```diff
@@ -252,17 +252,18 @@ struct InitDataOp<TurbineFast, SrcTrait>
             sz_info[1] = tf.to_cfd.fx_Len;
             sz_info[2] = tf.from_cfd.u_Len;
             sz_info[3] = tf.num_pts_tower;
+            sz_info[4] = tf.num_blade_elem;
         }
 
         // Broadcast data to everyone
         amrex::ParallelDescriptor::Bcast(
-            sz_info, 3, info.root_proc, tdata.tcomm);
+            sz_info.begin(), sz_info.size(), info.root_proc, tdata.tcomm);
```

Comment:
Should this be `sz_info.size()` or `sz_info.size()-1`?
Was this the bug that you had mentioned before?

Reply:
yes. It was broadcasting 3 values from the array, so not all the values in `sz_info`. Now it will always broadcast them all. Though.... now that I think about it. Maybe `tcomm` is of size 3? Ok need to check this.